src/Fields/Relationships/BelongsToMany.php

```diff
@@ -70,7 +70,7 @@ class BelongsToMany extends ModelRelationField implements
 
     protected string $inLineSeparator = '';
 
-    protected bool $inLineBadge = false;
+    protected Closure|bool|null $badgeCallback = null;
```

Comment:
```suggestion
    protected Closure|bool $inLineBadge = false;
```
а зачем нужен null? и я бы оставил название